lib/abs/icon_generator.dart

```diff
@@ -0,0 +1,90 @@
+import 'dart:io';
+
+import 'package:flutter_launcher_icons/flutter_launcher_icons_config.dart';
+import 'package:flutter_launcher_icons/logger.dart';
+
+///A base class to generate icons
```

Comment:
```suggestion
/// A base class to generate icons
```

Being picky with this one but space helps make it more readable